cmake/GetReaderWriterQueue.cmake

```diff
@@ -0,0 +1,18 @@
+include(FetchContent)
+
+FetchContent_Declare(
+  readerwriterqueue
+  GIT_REPOSITORY    https://github.com/cameron314/readerwriterqueue
+  GIT_TAG           v1.0.6
+)
+
+# TODO: Is this really still necessary? Or can we just use cmake 3.14?
```

Comment:
If we fix and update the rocky linux setup / test, we could require cmake 3.14 as minimum.

Reply:
Sounds great! I actually have no idea what's even the issue there. I might look into it in the near future.